### DISCUSSION TITLE
How to use sass instead of css

### DISCUSSION THREAD

**Discussion by jimmyff** [adityatelange/hugo-PaperMod#1031] [Q&A]
Hey, I'm utilising the `blank.css` override for some custom style stuff. I was hoping i could just create the equivalent `static/scss/extended/blank.scss` file and I'd get sassy css instead, but it doesn't seem to be getting picked up by the theme. Is that not a thing?

Thanks!

**Comment by adityatelange** [comment]
Hi SCSS is not supported, without hugo-extended support. We are not targeting hugo-extended hence that is not supported on Papermod as well.

**Reply by jimmyff** [reply]
Ah okay, thanks for letting me know.
